### ISSUE TITLE
Feature Request: Search Engine "Plugin" for the Google Chrome and the Mozilla Firefox

### ISSUE THREAD

**Issue by martinvahi** [privacore/open-source-search-engine#35]
In addition to the [Privafox](https://www.privacore.com/privafox/)_([archival copy](https://archive.is/yNKHK))_
the [findx](https://www.findx.com/)_([archival copy](https://archive.is/gakiT))_ might be integrated to the 
web browsers just like the [StartPage](https://www.startpage.com/)_([archival copy](https://archive.is/7UoMg))_ has been integrated.
The StartPage can be configured to be the browser's default search engine.

**Comment by brianschildt** [comment]
@martinvahi  - Thanks for the suggestion!  - I'm happy to tell you we just recently added our search extension for Chrome and the add-on for Firefox as well 
[Private search extension for Chrome](https://chrome.google.com/webstore/detail/findx/geahfdadhjfapiobjjgjdoimibohfeml)
[Private search add-on for Firefox](https://addons.mozilla.org/en-US/firefox/addon/findx/?src=search)

Extension for safari is on it's way - and very soon we will have more great news for Mobile, stay tuned.

**Comment by martinvahi** [comment]
Thanks :-)

**Comment by martinvahi** [comment]
I tried out the Chrome extension and as feedback I mention that
I find it to be **PROBABILISTICALLY LEAKING METADATA**, because if the findx main page
is loaded every time the Google Chrome opens a new tab and the WAN 
IP-address of the person's LAN does not change, then the findx.com can 
correlate the loading of the findx.com with the person's daily life, specially
if the person has a habit of powering off its laptop or 
closing all programs before logging off.

So, I switched off the extension.

I think that a proper way for the extension to behave is to just add the findx to the list of search engines. It should not do anything else.
